external/avro/src/main/scala/org/apache/spark/sql/avro/AvroDeserializer.scala

```diff
@@ -148,9 +148,9 @@ private[sql] class AvroDeserializer(
       }
 
       case (LONG, TimestampNTZType) => avroType.getLogicalType match {
-        // For backward compatibility, if the Avro type is Long and it is not logical type
-        // (the `null` case), the value is processed as timestamp without time zone type
-        // with millisecond precision.
+        // For keep consistent with TimestampLTZ, if the Avro type is Long and it is not
+        // logical type (the `null` case), the value is processed as timestamp without time
```

Comment:
Let's just say `TimestampNTZ` instead of `timestamp without time zone`, since you used `TimestampLTZ` at the beginning,